zmconfigd/smtpd_sender_restrictions.cf

```diff
@@ -2,7 +2,7 @@
 %%contains VAR:zimbraMtaSmtpdSenderRestrictions check_sender_access lmdb:/opt/zextras/conf/postfix_reject_sender%%
 %%contains VAR:zimbraServiceEnabled cbpolicyd^ check_policy_service inet:localhost:%%zimbraCBPolicydBindPort%%%%
 %%contains VAR:zimbraServiceEnabled amavis^ check_sender_access regexp:/opt/zextras/common/conf/tag_as_originating.re%%
-permit_mynetworks
+permit_mynetworks, reject_sender_login_mismatch
```

Comment:
This makes sense along with https://github.com/Zextras/carbonio-mailbox/pull/151/files.
One thing I wonder: does the attribute "zimbraMtaSmtpdSenderRestrictions" enable the loading of this configuration?
I thought the value here "reject_sender_login_mismatch" was supposed to be the value from "zimbraMtaSmtpdSenderRestrictions" attribute

Reply:
Thanks for the explanation! Yeah about server-level attribute they do not make sense anymore, there is no reason nowadays why one server should have a configuration different from global :)

Reply:
Luca Told me during testing that we will clean this in future when we will split mta.